schemas/data/intrumented-impressions.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/metrics/advertising/viewability/intrumented-impressions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "intrumented-impressions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metric"
+  ],
+  "description": "The advertising/viewability metric intrumented-impressions describes the whether the video impression had viewabilty library instrumented.",
```

Comment:
There is a typo, remove "the" after describes. 
I am not keen on the instrumented vs non instrumented nature of this. Can we represent this as an impression, and have how it was viewed as a property elsewhere?